### ISSUE TITLE
DASH

### ISSUE THREAD

**Issue by aw-west-defra** [Defra-Data-Science-Centre-of-Excellence/CDAP_training#18]
should this project be renamed DASH_training?

**Comment by Fraser-Buchanan** [comment]
Yes! You are correct. With the rebranding the documentation and github project should be renamed. It could also do with a bit or a restructure as the platform and documentation has developed. It will break some links but there is a push to update/improve the training documents. I'm going to leave this open as a reminder. It is coming...

**Comment by aw-west-defra** [comment]
[R's start-up message](https://github.com/Defra-Data-Science-Centre-of-Excellence/init-scripts/blob/5c9e2d84afc1605827df40c65d95620a0df1166f/src/base_libs.sh#L46) references this, keep us up-to-date with the correct link.
